readme.md

```diff
@@ -189,6 +189,21 @@ protected $dontKeepRevisionOf = ['category_id'];
 
 > The `$keepRevisionOf` setting takes precedence over `$dontKeepRevisionOf`
 
+### Storing additional fields in revisions
+
+In some cases, you'll want additional metadata from the models in each revision. An example of this might be if you 
+have to keep track of accounts as well as users. Simply create a migration to add the fields you'd like to your revision model,
+add them to your config/revisionable.php separated by pipes like so:
+
+```php 
+'additional_fields_in_model' => "account_id|permissions_id|other_id", 
+```
```

Comment:
Looks like this is incorrect, should be:

`additional_fields` and should accept an array